__tests__/creatures.test.js

```diff
@@ -0,0 +1,169 @@
+    },
+    {
+      url: 'https://www.shopmascot.com/image/cache/mascotnew/new196-800x800.jpg',
+      title: 'Rhino costume',
+      description: 'Mascots have to get their costumes somewhere',
+      keyword: 'rhino',
+      horns: 2
+    },
+    {
+      url:
+            'https://www.tinselbox.com/wp-content/uploads/2018/03/I-BELIEVE-IN-UNICORNS-FREE-PRINTABLE-WATERCOLOR-7-of-1.jpg',
+      title: 'Believe',
+      description: 'I believe in unicorns, do you?',
+      keyword: 'unicorn',
+      horns: 1
+    },
+    {
+      url:
+            'https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Markhor_Schraubenziege_Capra_falconeri_Zoo_Augsburg-02.jpg/220px-Markhor_Schraubenziege_Capra_falconeri_Zoo_Augsburg-02.jpg',
+      title: 'Markhor',
+      description:
+            'These wild goats eat snakes, then secrete a foam that locals fight over for the antivenom properties -- how cool is that?',
+      keyword: 'markhor',
+      horns: 2
+    },
+    {
+      url: 'http://www.zooborns.com/.a/6a010535647bf3970b0223c84d5959200c-800wi',
+      title: 'Baby markhor',
+      description: 'Even the babies are adorable',
+      keyword: 'markhor',
+      horns: 2
+    },
+    {
+      url:
+            'https://cbsnews3.cbsistatic.com/hub/i/r/2013/03/05/5b414225-a645-11e2-a3f0-029118418759/thumbnail/620x350/2d4cf24685b45c22912e64d2004fec8d/Baby_Mouflon_Wild_Sheep.jpg',
+      title: 'Baby mouflon',
+      description: 'The cuteness that is a baby mouflon asleep',
+      keyword: 'mouflon',
+      horns: 2
+    },
+    {
+      url:
+            'https://imgc.allpostersimages.com/img/print/posters/dlillc-jackson-s-chameleon_a-G-13448768-14258384.jpg',
+      title: 'Serious Jackson\'s Chameleon',
+      description: 'This one is very serious.',
+      keyword: 'chameleon',
+      horns: 3
+    },
+    {
+      url:
+            'https://www.nps.gov/band/learn/nature/images/short-horned-lizard-open-mouth-18.jpg?maxwidth=650&autorotate=false',
+      title: 'Horned Lizard',
+      description: 'Fave food: ants',
+      keyword: 'lizard',
+      horns: 100
+    },
+    {
+      url:
+            'https://upload.wikimedia.org/wikipedia/commons/thumb/d/df/Smaug_par_David_Demaret.jpg/290px-Smaug_par_David_Demaret.jpg',
+      title: 'Smaug',
+      description: 'Fan illustration of Smaug from \'The Hobbit\'',
+      keyword: 'dragon',
+      horns: 100
+    }
+  ];
+  
+  const response = await request.get('/api/creatures');
+  
+  expect(response.status).toBe(200); // 200 = OK
+  expect(response.body).toEqual(expected);
+  
+});
+
+test('GET /api/creatures/1', async () => {
+  const expected =  {
+    url:
+              'http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg',
+    title: 'UniWhal',
+    description: 'A unicorn and a narwhal nuzzling their horns',
+    keyword: 'narwhal',
+    horns: 1
+  };
+  
+  const response = await request.get('/api/creatures/:id');
```

Comment:
instead of :id you could put 1 or name or whatever. It is a variable, it just needs to match what you have in you GET route.